src/stores/Web3Store.js

```diff
@@ -3,6 +3,8 @@ import getWeb3, { getBalance, getWeb3Instance, getNetwork } from './utils/web3';
 import { balanceLoaded } from './utils/testUtils'
 import swal from 'sweetalert'
 import { BRIDGE_MODES } from './utils/bridgeMode'
+import { NoWallet } from '../components'
+import React from 'react'
```

Comment:
It seems that `NoWallet` and `React` are not used here. Also `swal` import is not needed anymore.

Reply:
Fix done.